packages/store-ui/src/atoms/Slider/Slider.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useCallback, useEffect, useRef, useState } from 'react'
+
+export type SliderProps = {
+  /**
+   * The minimum value of the slider.
+   */
+  min: number
+  /**
+   * The maximum value of the slider.
+   */
+  max: number
+  /**
+   * Show selected values.
+   *
+   * @default true
+   */
+  showValues?: boolean
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   *
+   * @default 'store-slider'
+   */
+  testId?: string
+  /**
+   * Callback that fires when the slider value changes.
+   */
+  onChange?: (value: { min: number; max: number }) => void
+  /**
+   * Set a human-readable name for the slider.
+   */
+  ariaLabel?: string
+  /**
+   * A function used to set a human-readable value text based on the slider's current value.
+   */
+  getAriaValueText?(value: number, thumb?: 'min' | 'max'): string
+}
+
+const Slider = ({
+  min,
+  max,
+  onChange,
+  testId = 'store-slider',
+  showValues = true,
+  ariaLabel,
+  getAriaValueText,
+}: SliderProps) => {
+  const [minVal, setMinVal] = useState(min)
+  const [maxVal, setMaxVal] = useState(max)
+
+  const minValRef = useRef(min)
+  const maxValRef = useRef(max)
+  const range = useRef<HTMLDivElement>(null)
+
+  const getPercent = useCallback(
+    (value) => Math.round(((value - min) / (max - min)) * 100),
+    [min, max]
+  )
+
+  useEffect(() => {
+    const minPercent = getPercent(minVal)
+    const maxPercent = getPercent(maxValRef.current)
+
+    if (range.current) {
+      range.current.style.left = `${minPercent}%`
+      range.current.style.width = `${maxPercent - minPercent}%`
+    }
+  }, [minVal, getPercent])
+
+  useEffect(() => {
+    const minPercent = getPercent(minValRef.current)
+    const maxPercent = getPercent(maxVal)
+
+    if (range.current) {
+      range.current.style.width = `${maxPercent - minPercent}%`
+    }
+  }, [maxVal, getPercent])
+
+  useEffect(() => {
+    onChange?.({ min: minVal, max: maxVal })
+  }, [minVal, maxVal, onChange])
+
+  return (
+    <div
+      data-store-slider
+      data-testid={testId}
+      aria-label={ariaLabel}
+      // eslint-disable-next-line jsx-a11y/role-has-required-aria-props
```

Comment:
Why is it okay to disable this rule?

Reply:
honestly, I can't understand this rule

Reply:
Yeah, it's a bit confusing 😅 
But as I understand it, every element that has an `aria-role` (built-in or not) needs to have a certain set of `aria-*` attributes.

In this case, since this `div` has a role of `slider`, so this `div` would need to have the attributes described in this example: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_slider_role#example_1_simple_numerical_range

Reply:
Uhmm, now looking at the example, I wonder if the best way to do this is actually removing this `role` from the `<div>`, since each of the inputs will already have their own built-in roles (and already have all necessary aria-attributes).

I know that this is getting more specifically into a11y, and since we decided to handle it in a future moment, we go ahead with the PR as is and then come back to this later :)

Reply:
Yes, I think we can discuss that in another pr.